app-text/jo/jo-1.6.recipe

```diff
@@ -0,0 +1,62 @@
+SUMMARY="JSON output from a shell"
+DESCRIPTION="jo is a small utility to create JSON objects on the command line.  It creates a JSON \
+string on stdout from words given it as arguments or read from stdin."
+HOMEPAGE="https://github.com/jpmens/jo"
+COPYRIGHT="2016-2022 Jan-Piet Mens"
+LICENSE="GNU GPL v2"
+REVISION="1"
+SOURCE_URI="https://github.com/jpmens/jo/archive/refs/tags/$portVersion.tar.gz"
+CHECKSUM_SHA256="ae2837ddad9993d5e28fe8db00efec6b2378d68b7366021353a1e297b5fa2025"
+PATCHES="$portName-$portVersion.patchset"
+
+ARCHITECTURES="all"
+
+PROVIDES="
+	jo = $portVersion
+	cmd:jo
+	"
+REQUIRES="
+	haiku
+	"
+
+BUILD_REQUIRES="
+	haiku_devel
+	"
+BUILD_PREREQUIRES="
+	cmd:aclocal
+	cmd:autoconf
+	cmd:autoheader
+	cmd:automake
+	cmd:autoreconf
+	cmd:awk
+	cmd:find
+	cmd:gcc
+	cmd:grep
+	cmd:install
+	cmd:make
+	cmd:pkg_config
+	cmd:sed
+	"
```

Comment:
Are all these needed? pkg-config for instance is only needed to find pandoc (which we don't have): https://github.com/jpmens/jo/blob/master/configure.ac#L44

Reply:
pkg-config can be removed but I believe the rest are needed.

Reply:
Actually, the build system requires pkg-config or it fails.  I can remove autoheader though.

Reply:
Haven't checked it (yet), thanks for looking into it :)